Makefile

```diff
@@ -108,8 +108,9 @@ else
 	$(call MINIKUBE_GO_BUILD_CMD,$@,$*)
 endif
 
-out/e2e-%-amd64:
-	GOOS=$* GOARCH=amd64 go test -c k8s.io/minikube/test/integration --tags=integration -o $@
+.PHONY: e2e-%-amd64
+e2e-%-amd64:
```

Comment:
Yeah sorry, I was more thinking of having two versions of the rule, one to run tests like we do for integration and one to build the binaries. This should be fine since it's only ever really used by CI, but I'm still not sure I follow why we have it as PHONY.